packages/bdd-tests/features/releasePrescription.feature

```diff
@@ -73,8 +74,46 @@ Feature: Releasing a prescription
       | statusReasonCode | statusReasonDisplay |
       | 0001             | Prescribing Error   |
     Then I get an error response 400
-      | message |
-      | Prescription/item was not cancelled. With dispenser. Marked for cancellation |
+      | message                                                                      | errorObject |
+      | Prescription/item was not cancelled. With dispenser. Marked for cancellation | entry       |
+    When I return the prescription
+      | statusReasonCode | statusReasonDisplay  |
+      | 0008                | Prescription expired |
+    Then I get a success response 200
+    Then the prescription is marked as To Be Dispensed
+
+  @excluded @AEA-2882
+  Scenario Outline: Return a repeat prescription
+    Given I create <number> prescription(s) for <dispensing site>
+      | prescriptionType | numberOfRepeatsAllowed |
+      | repeat           | 0                      |
+    When I release the prescriptions
+    Then I get <number> prescription(s) released to <dispensing site>
+    And the prescription is marked as With Dispenser
+    When I return the prescription
+      | statusReasonCode | statusReasonDisplay |
+      | <statusReasonCode>           | <statusReasonDisplay>     |
+    Then I get a success response 200
+    Then the prescription is marked as To Be Dispensed
+
+
+    Examples:
+      | number | dispensing site | statusReasonCode | statusReasonDisplay  |
+      #| 1      | FCG72           | 0004             | Another dispenser requested release on behalf of the patient |
+      | 1      | FCG71           | 0008             | Prescription expired |
+
+  @excluded @AEA-2882
```

Comment:
Why is this test excluded when it's one of the two main requirements on AEA-2882?